src/backend/wine/manager/downloader/__tests__/main/getter.test.ts

```diff
@@ -62,18 +62,4 @@ describe('Main - GetAvailableVersions', () => {
       )
     }
   })
-
-  test('Invalid repository key returns nothing', async () => {
-    axiosClient.get = jest.fn()
-
-    await expect(
-      getAvailableVersions({ repositorys: [-1] })
-    ).resolves.toStrictEqual([])
-
-    expect(axiosClient.get).not.toBeCalled()
-    expect(logWarning).toBeCalledWith(
-      'Unknown and not supported repository key passed! Skip fetch for -1',
-      'WineDownloader'
-    )
-  })
```

Comment:
The situation in this test can actually never happen in practice, since TS errors if you try to pass unknown values as enum keys
Not quite sure why it didn't before, but it definitely does now